server/controllers/projects.js

```diff
@@ -0,0 +1,18 @@
+const insertUserProjects = require('../database/queries/projects');
+
+const Projects = (req, res) => {
+	const user = req.body;
```

Comment:
make sure the userId is a number not string, I see it receives in the server as a string "userID" it should be a number